libs/api-services/src/lib/user/triggerPasswordReset.ts

```diff
@@ -0,0 +1,32 @@
+import { prisma } from '@dnd-assistant/prisma';
+import { PasswordResetMail } from '../mailer/mail/PasswordResetMail';
+import { generatePasswordResetSession } from '../session/generatePasswordResetSession';
+
+export const triggerPasswordReset = async (
+  email: string,
+  returnUrl: string
+) => {
+  const user = await prisma.user.findUniqueOrThrow({
+    where: {
+      email,
+    },
+  });
+
+  if (!user) {
```

Comment:
it throws above

Reply:
Ah yes, it should not throw above (see comment as to why)